docker/docker-compose-observability.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+
+volumes:
+  prometheus_data: {}
+
+services:
+  grafana:
+    image: grafana/grafana:latest
```

Comment:
I'm a bit sad that hadolint doesn't work for docker-compose file :(

Shouldn't we use a more precise tag ?